### DISCUSSION TITLE
Organised Def Hacks Worldwide 3.0

### DISCUSSION THREAD

**Discussion by avinashupadhya99** [EddieHubCommunity/support#1639] [Show and tell]
Hello EddieHub Community, 

Def Hacks Worldwide 3.0 is coming this June and will be from the 25 - 27th. A 36-hour adventure to create projects that can impact the world. It is open to high school and college/university students. No worries if you are a beginner with no experience. 

There will be workshops hosted for you to learn a new skill to create a new innovative project on the weekend. Turn your ideas into reality! There will be amazing prizes with more than $12,500 in cash prizes, internship opportunities, fun activities, mini-events, and opportunities to communicate with top tech minds. 

This year, our guest speakers include Viral Shah, co-founder of Julia Programming Language; Michael Siebel, co-founder of Y-Combinator; Josh Starmer, Founder of StatQuest; Eddie Jaoude, first Github star of the year. Join us and network with companies to expand your opportunities and get recognized.

 Sign up now at: https://forms.gle/5HsZmGf4qjS2aNZF7 
 Checkout our DevPost at https://defhacksworldwide.devpost.com/

 For more information, visit https://defhacks.co/hackathons/worldwide_3.0.html

LinkedIn Page: https://www.linkedin.com/company/def-hacks/
Instagram Page: https://www.instagram.com/def_hacks/
![Def_Hacks_W3 0_3](https://user-images.githubusercontent.com/52544819/118007073-6e654280-b369-11eb-8d82-d13be0cc24ad.png)

**Comment by kaiwalyakoparkar** [comment]
Will every attendee (Whoever makes a valid submission) get swags?

**Comment by avinashupadhya99** [comment]
Yes, we have plans for it. We are currently waiting for a couple of sponsors to confirm and hopefully we can make it happen. Also waiting for MLH to confirm as well. Fingers crossed, we should be able to. I will definitely update regarding this.

**Reply by kaiwalyakoparkar** [reply]
Sure thanks for confirming. Would love to hear from you as soon as everything finalizes!

**Comment by shikharcodes** [comment]
Good to know @avinashupadhya99, will take a look :D

**Reply by shikharcodes** [reply]
Will it be open innovation or there will be particular tracks?

**Comment by avinashupadhya99** [comment]
@shikharcodes it is open innovation. We will be adding a few sponsor tracks soon, the general tracks are open innovation

**Reply by shikharcodes** [reply]
Okay, thank you!

**Comment by kaiwalyakoparkar** [comment]
@avinashupadhya99 I would like to be a mentor at this hackathon. Hoping to hear back from you :smile:

**Reply by ghost** [reply]
Me too @avinashupadhya99?

**Reply by avinashupadhya99** [reply]
Hey, thank you for the interest! Can you please drop me a message on Discord (avinashupadhyaya#4664) and I can share the discord link along with setting y'all up as mentors :)
